packages/website/components/meta-context.js

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+
+const ContextMeta = React.createContext({});
```

Comment:
This is probably a good idea for all of our other generated data too, like the pages list

Reply:
Hmm. Probably true. I'm slightly worried about loading data into pages it's not needed (particularly as the pages list grows)

In part did this because of how little data it was, and because it is used at the top level

Reply:
the flip side is that if we add that data into __app or something then it will only be downloaded/bundled once and then can be re used across pages. Not sure if next gives us any optimisations on that though.

Reply:
the flip side is that if we add that data into __app or something then it will only be downloaded/bundled once and then can be re used across pages. Not sure if next gives us any optimisations on that though.